packages/terra-avatar/docs/upgrade-guide.md

```diff
@@ -1,4 +1,36 @@
 # terra-avatar Upgrade Guide
+## Changes from version 2 to version 3
+
+### Removed
+* `Shared User` subcomponent From Avatar
+
+### Added
+* the `generic` subcomponent that replaces the `sharedUser` subcomponent with a new `variant` prop that can be `single-user`, `shared-user`, or `provider`.
```

Comment:
```suggestion
* `generic` subcomponent  replaces the `sharedUser` subcomponent
* `generic` subcomponent contains a `variant` prop that toggles between `single-user`, `shared-user`, and `provider`.
```

Reply:
Thank you for the corrections..! updated [changle-log ](https://github.com/cerner/terra-core/blob/71741fd4912d5c77ebcf9d14de14ee575eb84690/packages/terra-avatar/CHANGELOG.md) like mentioned above